packages/client/src/mdx.tsx

```diff
@@ -18,77 +17,3 @@ export function MDX<TComponents extends Record<string, ComponentType<any>> = Rec
 }) {
   return <MDXRemote compiledSource={mdx.compiledSource} components={{ ...components, ...extraComponents }} />;
 }
-
-const protocols = ['http', 'https', 'mailto', 'tel'];
-
-function uriTransformer(uri?: string) {
-  const url = (uri || '').trim();
-  const first = url.charAt(0);
-
-  if (first === '#' || first === '/') {
-    return url;
-  }
-
-  const colon = url.indexOf(':');
-  if (colon === -1) {
-    return url;
-  }
-
-  let index = -1;
-
-  while (++index < protocols.length) {
-    const protocol = protocols[index];
-
-    if (colon === protocol.length && url.slice(0, protocol.length).toLowerCase() === protocol) {
-      return url;
-    }
-  }
-
-  index = url.indexOf('?');
-  if (index !== -1 && colon > index) {
-    return url;
-  }
-
-  index = url.indexOf('#');
-  if (index !== -1 && colon > index) {
-    return url;
-  }
-
-  return 'javascript:void(0)';
-}
-
-export const RemoteGHMarkdown = <TComponents extends Record<string, ComponentType<any>> = Record<string, never>>({
-  content,
-  repo,
-  directory,
-  extraComponents,
-}: {
-  content: CompiledMDX;
-  repo?: string;
-  directory?: string;
-  extraComponents?: ExtraMdxComponents<TComponents>;
-}) => {
-  return (
-    <MDX
-      mdx={content.mdx}
-      extraComponents={{
-        ...extraComponents,
-        img(props: ComponentProps<'img'>) {
-          let src = uriTransformer(props.src);
-
-          if (repo) {
-            let modified = repo.replace('https://github.com/', 'https://raw.githubusercontent.com/') + '/HEAD/';
-
-            if (directory) {
-              modified = modified + directory;
-            }
-
-            src = modified + (src.startsWith('.') ? src.substr(1) : src);
-          }
-
-          return <img {...props} alt={props.alt} src={src} />;
-        },
-      }}
-    />
-  );
-};
```

Comment:
seems like not used anywhere, dead code

Reply:
I thought this is what was used in `plugins` pages to grab the readme from the repo package and then show it on the page?

Reply:
It is used in envelop https://github.com/n1ru4l/envelop/blob/4eaaf76d418cf56476d57c176ca876fff89a86b2/website/src/pages/plugins/%5Bname%5D.tsx#L7